CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Enhancements:
 - Added trace logging level below Debug; maps to Trace in log4net/NLog, and Verbose in Serilog (@pi3k14, #404)
 - Recognize read-only parameters by the `In` modreq (@zvirja, #406)
 - DictionaryAdapter: Exposed GetAdapter overloads with NameValueCollection parameter in .NET Standard (@rzontar, #423)
+- New `IInvocation.GetProceedInfo()` method to enable better implementations of asynchronous interceptors (@stakx, #439)
```

Comment:
@stakx the method name in the changelog is out of date

Reply:
Another good catch... fixed. Say thanks to your hawk's eyes!